.gitignore

```diff
@@ -1,5 +1,7 @@
 # Dependency directory
 node_modules/
-
 # Generated by load-runner
 results-*
+.idea/
+raincatcher-demo-load-testing.iml
```

Comment:
If these .iml files appear in all repos that you'll touch, one thing you could consider is adding `*.iml` to a global gitignore file for your user on your machine, rather than adding a specific one for each repo.